src/pages/my_page/user_profile/MyProjectRegistrations.tsx

```diff
@@ -37,36 +27,25 @@ export const MyProjectRegistrations = ({
   });
 
   const projects = projectsQuery.data?.hits ?? [];
-  const filteredProjects = projects
-    .filter(
-      ({ status }) =>
-        (status === 'ACTIVE' && selectedOngoing) ||
-        (status === 'NOTSTARTED' && selectedNotStarted) ||
-        (status === 'CONCLUDED' && selectedConcluded) ||
-        ((status === 'ACTIVE' || status === 'NOTSTARTED' || status === 'CONCLUDED') &&
-          !selectedOngoing &&
-          !selectedNotStarted &&
-          !selectedConcluded)
-    )
-    .sort((a, b) => {
-      if (a.status === 'ACTIVE' && b.status !== 'ACTIVE') {
-        return -1;
-      } else if (a.status !== 'ACTIVE' && b.status === 'ACTIVE') {
-        return 1;
-      } else if (a.status === 'NOTSTARTED' && b.status !== 'NOTSTARTED') {
-        return -1;
-      } else if (a.status !== 'NOTSTARTED' && b.status === 'NOTSTARTED') {
-        return 1;
-      }
-      return new Date(a.startDate).getTime() - new Date(b.startDate).getTime();
-    });
+  const filteredProjects = projects.sort((a, b) => {
+    if (a.status === 'ACTIVE' && b.status !== 'ACTIVE') {
+      return -1;
+    } else if (a.status !== 'ACTIVE' && b.status === 'ACTIVE') {
+      return 1;
+    } else if (a.status === 'NOTSTARTED' && b.status !== 'NOTSTARTED') {
+      return -1;
+    } else if (a.status !== 'NOTSTARTED' && b.status === 'NOTSTARTED') {
+      return 1;
+    }
+    return new Date(a.startDate).getTime() - new Date(b.startDate).getTime();
+  });
 
   const projectsToShow = filteredProjects.slice(rowsPerPage * (page - 1), rowsPerPage * page);
   const validPage = page - 1 < Math.ceil(filteredProjects.length / rowsPerPage) ? page : 1;
 
   useEffect(() => {
```

Comment:
denne kunne nok vært tatt vekk nå, da den bare vil sette samme verdi som allerede er i state som default på onMount.
Tidligere var den her for å sende brukeren tilbake til første side når man endret filter, da det typisk ville endre antall treff sånn at man kunne stå på en side som ikke har noe data lenger